lib/segment/src/index/field_index/bool_index/mmap_bool_index.rs

```diff
@@ -202,12 +211,18 @@ impl MmapBoolIndex {
         !self.trues_slice.get(point_id as usize) && !self.falses_slice.get(point_id as usize)
     }
 
-    pub fn iter_values_map(&self) -> impl Iterator<Item = (bool, IdIter<'_>)> + '_ {
+    pub fn iter_values_map<'a>(
+        &'a self,
+        hw_counter: &'a HardwareCounterCell,
+    ) -> impl Iterator<Item = (bool, IdIter<'a>)> + 'a {
         [
             (false, Box::new(self.falses_slice.iter_trues()) as IdIter),
             (true, Box::new(self.trues_slice.iter_trues()) as IdIter),
         ]
         .into_iter()
+        .measure_hw_with_cell(hw_counter, size_of::<usize>(), |i| {
```

Comment:
why do we count usize for bool values?

Reply:
I confused underlying datatype (usize) with the actual bit type. Fixed now 👍🏻